code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1401,14 +1401,20 @@
 	. = ..()
 	L.add_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
 	ADD_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
-	ADD_TRAIT(L, TRAIT_SIXTHSENSE, type)
```

Comment:
how the hell did a gas that let you hear deadchat get approved in the first place?

Reply:
Not my finest moment